src/main/java/org/isf/examination/gui/PatientExaminationEdit.java

```diff
@@ -887,7 +887,7 @@ private CustomJDateChooser getJDateChooserDate() {
 
 	private VoLimitedTextArea getJTextAreaNote() {
 		if (jTextAreaNote == null) {
-			jTextAreaNote = new VoLimitedTextArea(300, 6, 20);
+			jTextAreaNote = new VoLimitedTextArea(2000, 6, 20);
```

Comment:
Would it make sense to have the value 2000 set as a public static variable in the persistence class and then used everywhere?    It would make the next change easier and ensure the exact same value is used.

Reply:
It is indeed also defined as JPA parameter:
```
	@Column(name="PEX_NOTE", length=2000)
	private String pex_note;
```
I was wondering if it can be used from there

Reply:
Done anyway, in combo with https://github.com/informatici/openhospital-core/pull/460

Thanks